testgres/node.py

```diff
@@ -340,11 +344,14 @@ def get_auth_method(t):
                 conf.write(u"fsync = off\n")
 
             # yapf: disable
-            conf.write(u"log_statement = {}\n"
-                       u"listen_addresses = '{}'\n"
-                       u"port = {}\n".format(log_statement,
-                                             self.host,
-                                             self.port))
+            conf.write(
+                u"log_statement = {}\n"
+                u"listen_addresses = '{}'\n"
+                u"port = {}\n"
+                u"max_replication_slots = {}\n".format(log_statement,
```

Comment:
IMHO `max_replication_slots` should be placed under `allow_streaming=True`, next to `max_wal_senders` etc.

Reply:
Agree as well